eda/src/autogluon/eda/visualization/jupyter.py

```diff
@@ -0,0 +1,39 @@
+from IPython.display import display, HTML, Markdown
+
+
+class JupyterMixin:
```

Comment:
Shall we add some docstring for this class?

Reply:
Removed the class - kept only Jupyter tools to fix Jupyter layouts.